patroni/ctl.py

```diff
@@ -454,7 +454,7 @@ def parse_scheduled(scheduled):
             raise PatroniCtlException(message.format(scheduled))
         return scheduled_at
 
-    return None
```

Comment:
Alright, alright. There could still be a solution which will satisfy all the parties involved. One moment, please.